os/arch/xtensa/src/esp32/esp32_timerisr.c

```diff
@@ -115,7 +115,7 @@ static inline uint32_t xtensa_getcompare(void)
 
```

Comment:
Is there a reason to change file permissions?

Reply:
As @junmin-kim said, please revert a change of file permission.

Reply:
reverted the file permission.